src/swarmus_ros_simulation/launch/reusable/simulation_services.launch

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" ?>
+<!-- This launch nodes that need to be started exactly once per simulation. These include:
+    - The Gazebo runtime
+    - The communication broker
+    - The tf publisher node
+-->
+<launch>
+    <!-- Gazebo -->
+    <include file="$(find gazebo_ros)/launch/empty_world.launch">
+        <arg name="debug" value="false" />
+        <arg name="gui" value="true" />
+        <arg name="paused" value="false"/>
+        <arg name="use_sim_time" value="true"/>
+        <arg name="headless" value="false"/>
+    </include>
+
+    <!-- Communication Broker -->
+    <rosparam command="load" file="$(find swarmus_ros_simulation)/launch/config/broker_config.yaml" />
+    <node name="commBroker" pkg="hive_connect" type="communication_broker"  output="screen" >
+        <!-- Parameter name to use to find the robot list. Should match the yaml config file -->
+        <param name="configList" value="/Broker/robots"/>
+    </node>
+
+    <!-- Interlocalisation -->
+    <node pkg="swarmus_ros_simulation" type="tf_publisher" name="tf_publisher"/>
```

Comment:
Actually it's not an interlocalization node but its needed to have interlocalization. It's more like a a tf publisher that links all the robots to a global frame. You could say "World frame publisher" and you could add a note that it's used by the simulated interlocalization nodes.